### PR TITLE
Bump to 2.0.0rc3

### PR DESCRIPTION
**Pull request by matham**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by matham** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -1,8 +1,9 @@
-if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
-  export USE_X11=1;
+if [ "$(uname)" == "Darwin" ]; then
+  export USE_X11=0;
   export USE_GSTREAMER=1;
+  export CPATH=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include
```
Comment (chrisburr) [inline recipe/build.sh:4]: See https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks

This file is available in the 10.10 SDK and you will need to rerender after making the changes suggested in the docs.

**Review by chrisburr — CHANGES_REQUESTED** [review]

File: recipe/build.sh (hunk 1 above)
Comment (matham) [reply recipe/build.sh:4]: Hmm, I made a new PR with the suggested changes https://github.com/conda-forge/kivy-feedstock/pull/21/files#diff-ff61408cdc05bc9667deeadb55e4aaceb1371972076b6bf6934f9008920f2bd2 but it still fails not finding the header. 

Are you sure 10.9 doesn't have `os/availability.h`, rather then it being a config issue? I don't easily have a mac to test on.

Comment (chrisburr) [reply recipe/build.sh:4]: Actually I was mixed up with `xpc`, `os/availability.h` first appears in `10.12`. From https://github.com/phracker/MacOSX-SDKs/:
```
$ find MacOSX-SDKs -name "availability.h" | sort -u
MacOSX-SDKs/MacOSX10.10.sdk/usr/include/xpc/availability.h
MacOSX-SDKs/MacOSX10.11.sdk/usr/include/xpc/availability.h
MacOSX-SDKs/MacOSX10.12.sdk/usr/include/os/availability.h
MacOSX-SDKs/MacOSX10.12.sdk/usr/include/xpc/availability.h
MacOSX-SDKs/MacOSX10.13.sdk/usr/include/os/availability.h
MacOSX-SDKs/MacOSX10.13.sdk/usr/include/xpc/availability.h
MacOSX-SDKs/MacOSX10.14.sdk/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/os/availability.h
MacOSX-SDKs/MacOSX10.14.sdk/usr/include/os/availability.h
MacOSX-SDKs/MacOSX10.14.sdk/usr/include/xpc/availability.h
MacOSX-SDKs/MacOSX10.15.sdk/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/os/availability.h
MacOSX-SDKs/MacOSX10.15.sdk/usr/include/os/availability.h
MacOSX-SDKs/MacOSX10.15.sdk/usr/include/xpc/availability.h
```

Comment (matham) [reply recipe/build.sh:4]: Thanks, that got me past this issue, but now there's a second sdk issue here: https://github.com/conda-forge/kivy-feedstock/runs/1257908137 for https://github.com/conda-forge/kivy-feedstock/pull/21. The error is:

```
building 'kivy.core.image.img_imageio' extension
x86_64-apple-darwin13.4.0-clang -fno-strict-aliasing -Wsign-compare -Wunreachable-code -DNDEBUG -fwrapv -O3 -Wall -Wstrict-prototypes -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O3 -pipe -fdebug-prefix-map=${SRC_DIR}=/usr/local/src/conda/${PKG_NAME}-${PKG_VERSION} -fdebug-prefix-map=/Users/runner/miniforge3/conda-bld/kivy_1602749736888/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac=/usr/local/src/conda-prefix -flto -Wl,-export_dynamic -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O3 -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -isystem /Users/runner/miniforge3/conda-bld/kivy_1602749736888/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/include -fdebug-prefix-map=/Users/runner/miniforge3/conda-bld/kivy_1602749736888/work=/usr/local/src/conda/kivy-2.0.0rc4 -fdebug-prefix-map=/Users/runner/miniforge3/conda-bld/kivy_1602749736888/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac=/usr/local/src/conda-prefix -D_FORTIFY_SOURCE=2 -mmacosx-version-min=10.9 -isystem /Users/runner/miniforge3/conda-bld/kivy_1602749736888/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/include -arch x86_64 -Ikivy/include -I/Users/runner/miniforge3/conda-bld/kivy_1602749736888/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/include/python3.8 -c kivy/core/image/img_imageio.c -o build/temp.macosx-10.9-x86_64-3.8/kivy/core/image/img_imageio.o -F/Applications/Xcode_11.7.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks
clang-10: warning: -Wl,-export_dynamic: 'linker' input unused [-Wunused-command-line-argument]
In file included from kivy/core/image/img_imageio.c:611:
In file included from /Applications/Xcode_11.7.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/CoreGraphics.framework/Headers/CGDataProvider.h:14:
/Applications/Xcode_11.7.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/CoreGraphics.framework/Headers/CGBase.h:321:78: warning: unknown attribute 'swift_name' ignored [-Wunknown-attributes]
typedef struct  CF_BRIDGED_TYPE(id) __IOSurface *IOSurfaceRef __attribute__((swift_name("IOSurfaceRef")));
                                                                             ^
In file included from kivy/core/image/img_imageio.c:611:
In file included from /Applications/Xcode_11.7.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/CoreGraphics.framework/Headers/CGDataProvider.h:15:
/Applications/Xcode_11.7.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CFURL.h:1159:83: error: expected ','
   kCFURLBookmarkCreationWithSecurityScope API_AVAILABLE(macos(10.7), macCatalyst(13.0))  API_UNAVAILABLE(ios, watchos, tvos) = ( 1UL << 11 ), // Mac OS X 10.7.3 and later, include information in the bookmark data which allows the same sandboxed process to access the resource after being relaunched
                                                                                 ^
/Applications/Xcode_11.7.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CFURL.h:1160:98: error: expected ','
    kCFURLBookmarkCreationSecurityScopeAllowOnlyReadAccess API_AVAILABLE(macos(10.7), macCatalyst(13.0)) API_UNAVAILABLE(ios, watchos, tvos) = ( 1UL << 12 ), // Mac OS X 10.7.3 and later, if used with kCFURLBookmarkCreationWithSecurityScope, at resolution time only read access to the resource will be granted
                                                                                                 ^
/Applications/Xcode_11.7.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CFURL.h:1170:85: error: expected ','
    kCFURLBookmarkResolutionWithSecurityScope API_AVAILABLE(macos(10.7), macCatalyst(13.0)) API_UNAVAILABLE(ios, watchos, tvos) = ( 1UL << 10 ), // Mac OS X 10.7.3 and later, use the secure information included at creation time to provide the ability to access the resource in a sandboxed process.
```

The closest related issue I can find is https://github.com/giampaolo/psutil/issues/1632 and there it seems to be a problem of multiple sdks on the system conflicting.

Comment (chrisburr) [reply recipe/build.sh:4]: It should be using the value of `CONDA_BUILD_SYSROOT` or `SDKROOT` though it's probably trying to be clever and use the value from xcode.

Comment (chrisburr) [reply recipe/build.sh:4]: Found it! https://github.com/kivy/kivy/blob/d74461b200a2182581f229082b041a4c70a673cb/setup.py#L606-L617

The cleanest solution is probably to make a PR upstream to respect the `SDKROOT` variable if set before falling back to the current method.

Comment (matham) [reply recipe/build.sh:4]: Thanks! I added a patch file to the PR and that fixed it. I'll make a PR upstream and then remove the patch when it's fixed!

**Comment by matham** [comment]
Closing in favor of https://github.com/conda-forge/kivy-feedstock/pull/21.

**Pull request closed without merging**